maps_dashboards/public/components/layer_control_panel/layer_control_panel.tsx

```diff
@@ -231,10 +238,10 @@ export const LayerControlPanel = memo(
       if (!selectedLayerConfig) {
         return;
       }
-      if (selectedLayerConfig.type === DASHBOARDS_MAPS_LAYER_TYPE.OPENSEARCH_MAP) {
-        return;
-      }
-      if (!selectedLayerConfig.source.indexPatternId) {
+      if (
+        selectedLayerConfig.type === DASHBOARDS_MAPS_LAYER_TYPE.OPENSEARCH_MAP ||
```

Comment:
i think we have to introduce a category inside Specification to differentiate types of layer instead of individual layer. Like: Data Layer / Basemap layer something like that

Reply:
Yeah. Some places need to distinguish data layer or reference layer. I defined a layerTypeLook, we maybe could use it.


```
if (referenceLayerTypeLookup[selectedLayerConfig.type]) {
  return;
}

export const referenceLayerTypeLookup = {
  [DASHBOARDS_MAPS_LAYER_TYPE.OPENSEARCH_MAP]: true,
  [DASHBOARDS_MAPS_LAYER_TYPE.CUSTOM_MAP]: true,
  [DASHBOARDS_MAPS_LAYER_TYPE.DOCUMENTS]: false,
};
```